LogicLayer/GameEventParticipations/GameEventParticipationService.cs

```diff
@@ -63,6 +64,29 @@ public async Task SendGameEventInvitationAsync(SendGameEventInvitationDto gameEv
             }
         }
 
+        public async Task SendGameEventInvitationAsync(IEnumerable<SendGameEventInvitationDto> gameEventInvitationDtos)
+        {
+            using (var transaction = _repository.BeginTransaction())
+            {
+                foreach (var gameEventInvitationDto in gameEventInvitationDtos)
+                {
+                    try
+                    {
+                        await SendGameEventInvitationAsync(gameEventInvitationDto);
+                    }
+                    catch (ArgumentOutOfRangeException exception)
+                    {
+                        throw exception;
+                    }
+                    catch
+                    {
+                    }
```

Comment:
You're catching all exceptions and not handling them. This code looks like Java.

Reply:
Marking as done.